yannickl88/features-bundle/1.0/etc/packages/features.yaml

```diff
@@ -0,0 +1,21 @@
+features:
+    tags:
+#        my.feature:
+#            request: ["beta", "on"]
+
+services:
```

Comment:
`container.yaml` is something that is part of the app? or is it something I need to add?